skopt/space/space.py

```diff
@@ -143,6 +144,10 @@ def __init__(self, low, high, prior="uniform", transform=None):
         self.high = high
         self.prior = prior
         self.transform_ = transform
+        if rvs == "uniform":
+            _rvs = uniform
+        else:
```

Comment:
can we make this explicit and switch to pseudo uniform only if `rvs == "pseudo-uniform"`?